content/en/news/releases/1.4.x/announcing-1.4.5/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Announcing Istio 1.4.5
+linktitle: 1.4.5
+subtitle: Patch Release
+description: Istio 1.4.5 patch release.
+publishdate: 2020-02-18
+release: 1.4.5
+aliases:
+    - /news/announcing-1.4.5
```

Comment:
I'm not sure of the reasons for this alias, and maybe it's something in the doc infrastructure that wants this link to also be active.

Reply:
Looks like it's a follow on to adding them in all announcements in https://github.com/istio/istio.io/pull/5426.